stable/kured/templates/daemonset.yaml

```diff
@@ -48,18 +52,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-          volumeMounts:
-            # Needed for two purposes:
-            # * Testing for the existence of /var/run/reboot-required
-            # * Accessing /var/run/dbus/system_bus_socket to effect reboot
-            - name: hostrun
-              mountPath: /var/run
       restartPolicy: Always
       {{- with .Values.tolerations }}
       tolerations:
 {{ toYaml . | indent 8 }}
       {{- end }}
-      volumes:
```

Comment:
isn't this still needed?

Reply:
hey @davidkarlsen 

this isn't needed anymore since the securityContext is used to nsenter /proc/1/ns/mnt. 
Have a look at this [commit](https://github.com/weaveworks/kured/commit/0cd450b7bc758de09fa90fcd34530159dc6be33d) - they also remove the volume mounts in their deployment